src/main/resources/presets/protocols/10x.yaml

```diff
@@ -86,154 +77,56 @@
-  refineTagsAndSort:
-    whitelists:
-      CELL: builtin:737K-august-2016
-    runCorrection: true
-    parameters:
-      correctionPower: 0.001
-      backgroundSubstitutionRate: 0.001
-      backgroundIndelRate: 1.0e-05
-      minQuality: 12
-      maxSubstitutions: 2
-      maxIndels: 2
-      maxTotalErrors: 3
-      postFilter:
-        type: filter_and
-        operands:
-          - type: filter_groups
-            groupingKeys:
-              - allTags:Cell
-              - allTags:Molecule
-            predicates:
-              - metrics:
-                  - type: "group_metric_sum_weight"
-                    reportHist:
-                      log: true
-                      binNumber: 0
-                      minBinWidth: 0.2
-                operator:
-                  type: group_operator_lowest_threshold
-                  operators:
-                    - type: group_operator_advanced_thresholding
-                      algo:
-                        type: otsu
-                      histTransformers:
-                        # Artificially adds 15% of singletons to the histogram before auto-thresholding algorithm
-                        - type: inflate
-                          fraction: 0.15
-                      logX: true
-                      minimalSample: 20
-                      fallbackThreshold: 1.0
-                    - type: group_operator_cumtop
-                      share: 0.85
-                      round: Down
           - type: filter_groups
             groupingKeys:
               - allTags:Cell
               - allTags:Molecule
             predicates:
               - metrics:
                   - type: group_metric_kmer_diversity
-                    sequences: [ "targets" ]
+                    sequences:
+                      - targets
                     k: 12
                     minimalQuality: 20
                     topShare: 0.7
                     reportHist:
+                      log: false
                       binNumber: 40
+                      minBinWidth: 0.0
+                      multiplyWeightByKey: false
                 operator:
                   type: group_operator_range
-                  lower: 200
-          - type: filter_groups
-            groupingKeys:
-              - allTags:Cell
-            predicates:
-              - metrics:
-                  - type: group_metric_nunique
-                    reportHist:
-                      log: true
-                      binNumber: 0
-                      minBinWidth: 0.2
-                    keys:
-                      - allTags:Molecule
-                operator:
-                  type: group_operator_range
-                  lower: 3.0
+                  lower: 150.0
   assemblePartial:
     inheritFrom: assemblePartial-base
   assemble:
     inheritFrom: assemble-with-consensus-base
     overrides:
       clnaOutput: true
+      cellLevel: true
+      consensusAssemblerParameters:
```

Comment:
Maybe move whole block to `assemble-with-consensus-base` ?

Reply:
Other single cell protocols, like BD, also use `assemble-with-consensus-base`, but not at cell level